### PR TITLE
User login register

### PR DESCRIPTION
**Pull request by ljubar**
| Q                         | A
| ------------------------- | ---
| Bug fix?                  | |no
| New feature?              | yes
| Changelog update required | yes
| Fixed tickets             | SWP-765
| License                   | AGPLv3

### CODE REVIEW

File: src/SWP/Bundle/FixturesBundle/Resources/themes/DefaultTheme/FOSUserBundle/views/Registration/check_email.html.twig [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{% extends "@FOSUser/layout.html.twig" %}
+
+{% trans_default_domain 'FOSUserBundle' %}
+
+{% block fos_user_content %}
+<style>
+	.fosLogin {
+		display: none;
```
Comment (ahilles107) [inline src/SWP/Bundle/FixturesBundle/Resources/themes/DefaultTheme/FOSUserBundle/views/Registration/check_email.html.twig]: one tab not needed

File: src/SWP/Bundle/FixturesBundle/Resources/themes/DefaultTheme/FOSUserBundle/views/Registration/check_email.html.twig [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+	}
+</style>
+    <p>{{ 'registration.check_email'|trans({'%email%': user.email}) }}</p>
```
Comment (ahilles107) [inline src/SWP/Bundle/FixturesBundle/Resources/themes/DefaultTheme/FOSUserBundle/views/Registration/check_email.html.twig]: tab not needed

File: src/SWP/Bundle/FixturesBundle/Resources/themes/DefaultTheme/FOSUserBundle/views/layout.html.twig [hunk 3]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
```
Comment (ahilles107) [inline src/SWP/Bundle/FixturesBundle/Resources/themes/DefaultTheme/FOSUserBundle/views/layout.html.twig]: we should extend here main theme layout (not create new one)

File: src/SWP/Bundle/FixturesBundle/Resources/themes/DefaultTheme/public/css/static.css [hunk 4]
```diff
@@ -81,7 +80,7 @@
 .icon_padlock,
 .icon_key
 {
-	border-right: 1px solid #e8e8e8;
+	/* border-right: 1px solid #e8e8e8; */
```
Comment (ahilles107) [inline src/SWP/Bundle/FixturesBundle/Resources/themes/DefaultTheme/public/css/static.css]: remove comment

File: src/SWP/Bundle/FixturesBundle/Resources/themes/DefaultTheme/views/_tpl/login-popup.html.twig [hunk 5]
```diff
@@ -0,0 +1,154 @@
+            <div class="container">
```
Comment (ahilles107) [inline src/SWP/Bundle/FixturesBundle/Resources/themes/DefaultTheme/views/_tpl/login-popup.html.twig]: indentation looks very bad here

File: src/SWP/Bundle/FixturesBundle/Resources/themes/DefaultTheme/FOSUserBundle/views/layout.html.twig (hunk 3 above)
Comment (ljubar) [inline src/SWP/Bundle/FixturesBundle/Resources/themes/DefaultTheme/FOSUserBundle/views/layout.html.twig]: it is for popup window; for confirmation link target we extend our theme's base.html.twig

File: src/SWP/Bundle/FixturesBundle/Resources/themes/DefaultTheme/views/_tpl/login-popup.html.twig [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+                       #}
+                       <input type="hidden" name="_login_success_path" value="{{ url(homepage) }}" />
+                       <input type="hidden" name="_login_failure_path" value="{{ url(homepage) }}" />
+                       <input type="submit" value="Sign In" class="float_right">
+                     </div>
+                   </div>
+
+                   </form>
+
+                   <h4>Don't have account yet? Create one here:</h4>
```
Comment (takeit) [inline src/SWP/Bundle/FixturesBundle/Resources/themes/DefaultTheme/views/_tpl/login-popup.html.twig]: some strings in that file are not translated

**Pull request merged**